packages/react-components/src/atoms/Pill.tsx

```diff
@@ -6,7 +6,8 @@ import { lineHeight, perRem } from '../pixels';
 
 const borderWidth = 1;
 const styles = css({
-  display: 'inline-block',
+  display: 'flex',
+  alignItems: 'center',
```

Comment:
I think we need to constraint the width. It's looking like this on News
<img width="1439" alt="Screenshot 2023-09-12 at 08 15 55" src="https://github.com/yldio/asap-hub/assets/16595804/f080f4c0-9454-4bb6-b8a2-7ff997c68600">